wemake_python_styleguide/visitors/ast/complexity/offset.py

```diff
@@ -23,6 +23,8 @@
     'visit_Break',
     'visit_Assign',
     'visit_Expr',
+    'visit_AsyncFor',
```

Comment:
Sorry, but  it seems that I have forgotten about `visit_Await`. Can you please add it here and write a unit test for `await something()`?